### PR TITLE
fix: remove unused var

### PR DESCRIPTION
**Pull request by poolpitako**

### CODE REVIEW

File: contracts/BaseStrategy.sol [hunk 1]
```diff
@@ -624,7 +624,6 @@ abstract contract BaseStrategy {
         // We usually don't need tend, but if there are positions that need
         // active maintainence, overriding this function is how you would
         // signal for that.
-        uint256 callCost = ethToWant(callCostInWei);
```
[the remark this answers is not in the document]
Comment (poolpitako) [reply contracts/BaseStrategy.sol:627]: Perhaps adding it inside the comment block?

[the remark this answers is not in the document]
Comment (fubuloubu) [reply contracts/BaseStrategy.sol:627]: Sure, that works

[the remark this answers is not in the document]
Comment (steffenix) [reply contracts/BaseStrategy.sol:627]: If this is optional it should also be removed from https://github.com/yearn/yearn-vaults/blob/master/contracts/BaseStrategy.sol#L501

**Review by fubuloubu — APPROVED** [review]

File: contracts/BaseStrategy.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (fubuloubu) [reply contracts/BaseStrategy.sol:627]: It really isn't optional, it just doesn't make sense for our test strategy

**Pull request merged**